contracts/components/staking/FortaStaking.sol

```diff
@@ -353,6 +353,9 @@ contract FortaStaking is BaseComponentUpgradeable, ERC1155SupplyUpgradeable, Sub
         uint256 stakeValue = activeSharesToStake(activeSharesId, activeShares);
         uint256 inactiveShares = stakeToInactiveShares(FortaStakingUtils.activeToInactive(activeSharesId), stakeValue);
 
+        if (getSubjectTypeAgency(subjectType) == SubjectStakeAgency.DELEGATED || getSubjectTypeAgency(subjectType) == SubjectStakeAgency.DELEGATOR) {
```

Comment:
Please define a variable for `getSubjectTypeAgency(subjectType)`.

Reply:
I defined it in every instance where is at least 2 `getSubjectTypeAgency(subjectType)`